src/mesh_sandbox/store/base.py

```diff
@@ -72,6 +72,12 @@ class Store(ABC):
     def __init__(self, config: EnvConfig):
         self.config = config
 
+    async def reinitialise(self, clear_disk: bool):
```

Comment:
do we need this here? why not just have the method in File_store and memory_store?